app/js/lib/foxbox/foxbox.js

```diff
@@ -597,8 +429,8 @@ export default class Foxbox extends Service {
     let payload = { id: operation.id };
 
     if (operation.kind.typ === 'Binary') {
-      return fetchBlob(
-        `${this.origin}/api/v${settings.apiVersion}/channels/get`,
+      return this[p.net].fetchBlob(
+        `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/channels/get`,
```

Comment:
BTW, the line lengths limit is already enforced in jshint, but because of the transpilation to es5, the generated lines can be longer than the ones in the initial es6 code. So this line length enforcement ends up being rather useless.

Maybe we need to update the workflow to enforce coding style like that before the transpilation. I don't know if jshint or eshint supports the es6 syntax though.